### DISCUSSION TITLE
How to disable the default tab cmd.exe opens ?

### DISCUSSION THREAD

**Discussion by MyBlood** [Eugeny/tabby#9592] [Q&A] [answered]
Greetings, friends!

I can't find in the settings where it is regulated.

How can I make these tabs not open all the time? Sometimes one opens, sometimes three at once....

![image](https://github.com/Eugeny/tabby/assets/20350848/06bf30b0-a0b2-4f1c-87c1-19fc2f0182cb)

**Comment by Eugeny** [comment, marked as answer]
Settings -> Terminal -> Startup
